pkg/controller/lifecycle/actuator_test.go

```diff
@@ -49,49 +51,69 @@ var _ = Describe("Actuator", func() {
 
 	Context("getShootResources", func() {
 		const (
-			shootNamespace                = "garden-foo"
-			extensionNamespace            = "shoot--foo--bar"
-			shootAccessServiceAccountName = "extension-shoot-lakom-service-access"
+			shootNamespace                  = "garden-foo"
+			extensionNamespace              = "shoot--foo--bar"
+			scope                           = lakom.KubeSystemManagedByGardener
+			shootAccessServiceAccountName   = "extension-shoot-lakom-service-access"
+			validatingWebhookKey            = "validatingwebhookconfiguration____gardener-extension-shoot-lakom-service-shoot.yaml"
+			mutatingWebhookKey              = "mutatingwebhookconfiguration____gardener-extension-shoot-lakom-service-shoot.yaml"
+			roleKey                         = "role__kube-system__gardener-extension-shoot-lakom-service-resource-reader.yaml"
+			roleBindingKey                  = "rolebinding__kube-system__gardener-extension-shoot-lakom-service-resource-reader.yaml"
```

Comment:
This test was reworked with #109 as the rendered managed resources are now compressed with Brotli and put into single manifest, hence the secret data keys are not used anymore.  I guess something went wrong when the conflicts were resolved after the main branch was merged into the PR branch.